config.js

```diff
@@ -496,6 +496,14 @@ var config = {
     // and microsoftApiApplicationClientID
     // enableCalendarIntegration: false,
 
+    // Configs for prejoin page.
+    // prejoinConfig: {
+    //     // Replaces prejoinPageEnabled config.
+    //     enabled: true,
+    //     // List of buttons to hide from the extra join options dropdown.
+    //     hideExtraJoinButtons: ['no-audio', 'by-phone']
+    // },
+
     // When 'true', it shows an intermediate page before joining, where the user can configure their devices.
```

Comment:
Can you remove this entry?

Reply:
done. also moved the comment on the 'enabled' flag instead.